src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/JobScheduler.scala

```diff
@@ -48,6 +49,7 @@ class JobScheduler @Inject()(sensors: Sensors, executors: Executors, dagInstance
 
   def startManager(): Unit = {
     if(!isManagerRunningAtomic.get() && runningScheduler.isCompleted) {
+      TimeSensorQuartzSchedulerManager.start()
```

Comment:
Last thing, let's move this to Sensors to a method that will start all required dep. and here will be used only that method.